### PR TITLE
DOC: new doc cover

### PR DESCRIPTION
**Pull request by onesuper**

### CODE REVIEW

File: doc/source/locale/ja_JP/LC_MESSAGES/index.po [hunk 1]
```diff
@@ -215,3 +139,176 @@ msgstr ""
 #~ "<https://github.com/xorbitsai/inference/pull/402>`_"
 #~ msgstr ""
 
+#~ msgid "Xorbits Inference: Model Serving Made Easy🤖"
```
Comment (qinxuye) [inline doc/source/locale/ja_JP/LC_MESSAGES/index.po:142]: For this comment messages, I think we can remove them.

Comment (qinxuye) [reply doc/source/locale/ja_JP/LC_MESSAGES/index.po:142]: Let's do it later.

**Review by qinxuye — APPROVED** [review]
LGTM

File: doc/source/locale/ja_JP/LC_MESSAGES/index.po (hunk 1 above)
Comment (onesuper) [reply doc/source/locale/ja_JP/LC_MESSAGES/index.po]: Cleaned

**Pull request merged**